tasks/49_cat-and-dog-years/cat-and-dog-years.js

```diff
@@ -0,0 +1,37 @@
+function humanYearsCatYearsDogYears(years) {
+  return [years,catAge(years),dogAge(years)];
+}
+
+function catAge(years) {
+  let age = 0;
+  for (let i = 1; i <= years; i++) {
+    if (i === 1){
+      age += 15;
+      continue;
```

Comment:
Please avoid `continue` keywords in your code. It could be done cleaner. First of all task condition says that minimum human age will be 1 so there's no need to start counting from 0.